docs/articles/audio/voicenext/receive.md

```diff
@@ -0,0 +1,101 @@
+---
+uid: voicenext_receive
+title: Receiving
+---
+
+## Receiving with VoiceNext
+
+### Enable Receiver
+Receiving incoming audio is disabled by default to save on bandwidth, as most users will never make use of incoming data.
+This can be changed by providing a configuration object to `DiscordClient#UseVoiceNext()`.
+```cs
+var discord = new DiscordClient();
+
+discord.UseVoiceNext(new VoiceNextConfiguration()
+{
+    EnableIncoming = true
+});
+```
+
+### Establish Connection
+The voice channel join process is the exact same as when transmitting.
+```cs
+DiscordChannel channel;
+VoiceNextConnection connection = await channel.ConnectAsync();
+```
+
+### Write Event Handler
+We'll be able to receive incoming audio from the `VoiceReceived` event fired by `VoiceNextConnection`.
+```cs
+connection.VoiceReceived += ReceiveHandler;
+```
+
+Writing the logic for this event handler will depend on your overall goal. 
+
+The event arguments will contain a PCM audio packet for you to make use of.
+You can convert each packet to another format, concatenate them all together, feed them into an external program, or process the packets any way that'll suit your needs.
+
+When a user is speaking, `VoiceReceived` will be fired once every twenty milliseconds and its packet will contain twenty milliseconds worth of audio.
```

Comment:
Not necessarily true. Various clients tend to have wildly different settings, and the format doesn't even stay consistent throughout the transmission.